projects/toggle/src/lib/directives/toggle-group.directive.spec.ts

```diff
@@ -99,9 +99,9 @@ describe('KcToggleGroupDirective', () => {
     const onTouched = jasmine.createSpy();
     directive.toggleGroup['_onTouch'] = onTouched;
 
```

Comment:
We set `control = new FormControl('a');`, so selecting 'a' again here would result in 'untoggling'.